src/System.Private.CoreLib/shared/System/Span.Fast.cs

```diff
@@ -172,9 +172,7 @@ internal Span(ref T ptr, int length)
         {
             get
             {
-                int start = range.Start.FromEnd ? _length - range.Start.Value : range.Start.Value;
-                int end = range.End.FromEnd ? _length - range.End.Value : range.End.Value;
-                return Slice(start, end - start);
+                return Slice(range);
```

Comment:
Nit: => syntax... will stop commenting on these

Reply:
Thanks, I'll review all places and fix them.